src/explore-education-statistics-frontend/server.js

```diff
@@ -68,6 +68,37 @@ async function startServer() {
 
   const server = express();
 
+  function replaceLastOccurance(input, pattern, replacement) {
+    if (
+      input === undefined ||
+      input === null ||
+      input.length === 0 ||
+      !input.endsWith(pattern)
+    ) {
+      return input;
+    }
+
+    return `${input.slice(0, -pattern.length)}${replacement}`;
+  }
+
+  // Redirect URLs with trailing slash to equivalent without slash with 301
+  server.use((req, res, nextNotShadowed) => {
+    let newUri = req.url;
+    newUri = replaceLastOccurance(newUri, '/', '');
+    newUri = replaceLastOccurance(newUri, '/meta-guidance', '/data-guidance');
+    newUri = replaceLastOccurance(
+      newUri,
+      '/download-latest-data',
+      '/data-catalogue',
+    );
+
+    if (newUri !== req.url && newUri !== '') {
+      return res.redirect(301, newUri);
+    }
+    nextNotShadowed();
```

Comment:
If anyone can come up with a better name for this, be my guest. 

This method is ExpressJS' callback to just "go onto the next thing", which they call `next` by default. This is however shadowed if you import `next` (as in NextJS) above! I considered `continue` or `nextMiddleware` or `proceed` but for obvious reasons they're also problematic 🤷